fs/rc/rcserver/rcserver_test.go

```diff
@@ -847,3 +847,22 @@ func TestServeModTime(t *testing.T) {
 	}}
 	testServer(t, tests, &opt)
 }
+
+func TestContentTypeJSON(t *testing.T) {
+	tests := []testRun{
+		{
+			Name:        "Check Content-Type for JSON response",
+			URL:         "rc/noop", // Assuming rc/noop returns JSON
```

Comment:
You should be able to test the error case with `rc/error` - I'd suggest another test for that.

Reply:
Ok I added another test case for rc/error, good chance I did something in an odd way, lmk